pkgs/servers/sql/mariadb/default.nix

```diff
@@ -101,6 +101,15 @@ common = rec { # attributes common to both builds
 
   passthru.mysqlVersion = "5.7";
 
+  passthru.tests = {
+    mariadb-galera-mariabackup = nixosTests.mariadb-galera-mariabackup;
```

Comment:
YOu also need to add `nixosTests` as an argument.

Reply:
Fixed.